examples/Console/TestMetrics.cs

```diff
@@ -116,80 +115,51 @@ internal static object Run(MetricsOptions options)
                 });
             }
 
-            var cts = new CancellationTokenSource();
+            System.Console.WriteLine("Press any key to exit.");
```

Comment:
Curious, would we want "any key" or "ESC"? https://github.com/open-telemetry/opentelemetry-dotnet/blob/6e31ab66d2610216c3bae2cdbf9fc8c4d6aca0d1/test/OpenTelemetry.Tests.Stress/Skeleton.cs#L124

I noticed that sometimes folks pressed Enter twice and the application started then exited.

Reply:
will revisit this together with Prometheus example (it also uses Any Key today)

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/2983